gamefixes-steam/256330.py

```diff
@@ -0,0 +1,15 @@
+""" WRC 4
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    #Black screen without it
+    util.protontricks('d3dx9_42')
+    util.protontricks('d3dx9_43')
+    util.protontricks('faudio')
+    util.protontricks('xact')
+
+    # Audio breaks above 60 fps
+    util.set_environment('DXVK_FRAME_RATE', '60')
```

Comment:
Instead of this fix, a PR should be made for this upstream in DXVK.

Reply:
audio bug is not linux related, should I still ask DXVK to limit framerate upstream?

Reply:
Yes, please make a PR to DXVK as everyone will benefit and you're certain that playing the game above 60+ FPS will break audio. Also, I'm not even too sure we allow fixes to set this variable as this fix would be only one that sets this environment variable in this project.